.gitlab-ci.yml

```diff
@@ -59,20 +62,32 @@ build_doc:
       - .tox/
       - doc/build/
 
-RPM:
+rpm:
   stage: package
   image: fedora:latest
-  before_script:
-    - dnf --refresh -y install mock perl-TimeDate which make gzip tar perl git
+  <<: *install_system_deps
   script:
+    - dnf install -y --refresh mock perl-TimeDate gzip tar perl git
```

Comment:
Do we need `perl-TimeDate` anymore?

Reply:
don't think so, I deleted it, but added dateutil which is not in core